SwiftDungeon/GameManager.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  GameManager.swift
+//  SwiftDungeon
+//
+//  Created by Toro Juan D. on 3/12/18.
+//  Copyright © 2018 Toro Juan D. All rights reserved.
+//
+
+import Foundation
+import GameKit
+
+class GameManager {
```

Comment:
When you create a class try and think about the responsibilities of that class and name it accordingly. After you've written the class, look over what it does and ask yourself if the name is good.